### ISSUE TITLE
Error: Pack node (stack_24) axis attribute is out of bounds: 1

### ISSUE THREAD

**Issue by tessjansen95** [jfkirk/tensorrec#145]
Hi! My apologies if this has already been marked as an issue, I am getting this error every time I run getting_started.py:

`  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
2019-06-13 18:34:15.847329: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node gradients_3/SparseTensorDenseMatMul_11/SparseTensorDenseMatMul_grad/strided_slice. Error: Pack node (stack_24) axis attribute is out of bounds: 1
2019-06-13 18:34:15.847354: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node gradients_3/SparseTensorDenseMatMul_11/SparseTensorDenseMatMul_grad/strided_slice_1. Error: Pack node (stack_24) axis attribute is out of bounds: 1
2019-06-13 18:34:15.847387: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node gradients_3/SparseTensorDenseMatMul_12/SparseTensorDenseMatMul_grad/strided_slice. Error: Pack node (stack_23) axis attribute is out of bounds: 1
2019-06-13 18:34:15.847399: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node gradients_3/SparseTensorDenseMatMul_12/SparseTensorDenseMatMul_grad/strided_slice_1. Error: Pack node (stack_23) axis attribute is out of bounds: 1`

The issue seems to be explicitly with the **Arithmetic Optimizer**, as I receive the same error at each of the training steps, but with a different pack node stack error each time. 

I haven't made much any modifications to the code, and I have installed all dependencies. Some internet detective work has led me to believe this isn't anything kernel-breaking, as "TF will simply execute the un-optimized (but functionally correct) graph." But I would still like to witness this rec system at full capacity, so any help would be appreciated! Thank you :)

-Tess

**Comment by tessjansen95** [comment]
After playing around with it a bit, it appeared Tensorflow version 1.13 was causing the issues. I downgraded to 1.12 (after [finding this discussion](https://www.bountysource.com/issues/69430304-training-error-failed-to-run-optimizer-stage-removestackstridedslicesameaxis)) and that seemed to have solved the issue. Hopefully this will help other people as well who are experiencing a similar issue!
